### PR TITLE
[Fix] Flashbangs not affecting holder if in container

### PR DESCRIPTION
**Pull request by Ranged**
**What does this PR do:**
Flashbangs previously did not affect anybody on the same tile if it was inside a container, since the mobs on the tile would have the tile as loc and the flashbang would have the container as loc.

Using get_turf() fixes this.

Using git for the first time has been quite the adventure and I don't know how to get rid of those two junk commits that basically just undo each other.

**Changelog:**
:cl:
fix: fixed flashbangs not properly working inside containers
/:cl:

fixes:#10470

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
If you add `fixes:#10470` to your description it will automatically close that issue when this is merged

**Comment by Ranged** [comment]
> If you add `fixes:#10470` to your description it will automatically close that issue when this is merged

Neat, thank you.

File: code/game/objects/items/weapons/grenades/flashbang.dm [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
 
 //Bang
-	if((loc == M) || loc == M.loc)//Holding on person or being exactly where lies is significantly more dangerous and voids protection
+	if((get_turf(M)) == (get_turf(src)))//Holding on person or being exactly where lies is significantly more dangerous and voids protection
```
Comment (Citinited) [inline code/game/objects/items/weapons/grenades/flashbang.dm]: The set of brackets around each `get_turf()` are unnecessary

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 2]
```diff
@@ -737,6 +737,10 @@
 	var/needs_update = M.mutations.len > 0 || M.disabilities > 0
 
 	if(needs_update)
+		var/mut
```
Comment (KasparoVy) [inline code/modules/reagents/chemistry/reagents/medicine.dm]: What's this got to do with holding flashbangs? :thinking:

**Comment by Ranged** [comment]
Uhhh

This is me not understanding how2github. Specifically branches. But I know what went wrong, I'll just close this, fix my stuff and start anew.

I went around fixing some bugs and doing easy changes to get used to DM but messed up branching and the hole I've dug myself in is getting deeper

**Pull request closed without merging**